fastlane/lanes/release.rb

```diff
@@ -1,21 +1,15 @@
 # frozen_string_literal: true
 
 platform :android do
-  #####################################################################################
-  # code_freeze
-  # -----------------------------------------------------------------------------------
-  # This lane executes the steps planned on code freeze
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane code_freeze [update_release_branch_version:<update flag>] [skip_confirm:<skip confirm>]
+  # Creates a new release branch from the current trunk for code freeze.
   #
-  # Example:
-  # bundle exec fastlane code_freeze
-  # bundle exec fastlane code_freeze update_release_branch_version:false
-  # bundle exec fastlane code_freeze skip_confirm:true
-  #####################################################################################
-  desc 'Creates a new release branch from the current trunk'
-  lane :code_freeze do |options|
+  # @param [Boolean] skip_confirm Whether to skip the confirmation prompt
+  #
+  # @example
+  #   bundle exec fastlane code_freeze
+  #   bundle exec fastlane code_freeze skip_confirm:true
+  #
+  lane :code_freeze do |skip_confirm: false|
```

Comment:
Interesting to see this project runs `code_freeze` in one step instead of the `start_code_freeze` and `complete_code_freeze` like the rest. 

Maybe it hasn't been updated yet? Or are there some limitations to adopting the pattern?

Not suggesting to address this now, to be clear. Just flagging it.

Reply:
Yeah, there's also a few more non-standard things, like having all pipeline files under `.buildkite/` instead of `.buildkite/release-pipelines/` like the other repos.
I'll create another PR and diff as a follow up at some point, but this has to be done carefully as they need to land at the same time and any eventual already existing releases need to be reset so that the right pipelines are used.

Reply:
Sounds good.